src/components/MegaMenu/MegaMenuItem.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div
+    :class="[
+      {'pb-2': small},
+      {'pb-4': !small}
+    ]"
+  >
+    <component
+      :is="isExternal ? 'a' : 'router-link'"
+      :[linkProp]="to"
+      class="flex pt-1 flex-nowrap items-center hover:text-primary-500"
+    >
+      <img
+        :src="imageSource"
+        alt=""
+        :class="[
+          'self-center transition-transform duration-200 ease-linear transform hover:scale-110',
+          {'w-10 h-10': small},
+          {'w-12 h-12': !small}]"
+      >
+      <div class="pl-2 text-gray-900">
+        <span
+          :class="['pt-1 opacity-75 transition-colors transition-opacity no-underline md:pb-0.5 md:text-lg block relative align-top mx-auto whitespace-nowrap border-none text-normal text-small hover:text-primary-500', {'py-1': small}]"
+        >
+          <slot />
+        </span>
+        <div
+          v-if="subtitle"
+          class="opacity-80 pr-0 text-gray-700 text-sm hidden md:block"
+        >
+          {{ subtitle }}
+        </div>
+      </div>
+    </component>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'MegaMenuItem',
+  props: {
+    to: {
+      type: String,
+      required: true
+    },
+    imageSource: {
+      type: String,
+      required: true
+    },
+    subtitle: {
+      type: String,
+      default: ''
+    },
+    small: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    isExternal () {
+      const protocolRelativePattern = /^https?:\/\/|^\/\//i;
+      return protocolRelativePattern.test(this.to);
+    },
+    linkProp () {
+      return this.isExternal ? 'href' : 'to';
+    }
+  }
+};
+</script>
+
+<style scoped>
+  .text-small {
```

Comment:
Oop, we don't need this. We can replace with Tailwind's `text-lg` and it doesn't have to be important

Reply:
The thing is, though [Tailwind's documentation](https://tailwindcss.com/docs/font-size) states that `text-lg { font-size: 1.125rem ... }`
I'm getting: 
![image](https://user-images.githubusercontent.com/83967528/121357618-5224d900-c908-11eb-81a2-159382588255.png)
!